curiouslearning/FeedTheMonsterJS
# Fix CR-67: Added Tigragna Feedback audios

# Changes
- Added Tigragna feedback audios

# How to test
- Check for the feedback audios and text.

Ref: [CR-67](https://curiouslearning.atlassian.net/jira/software/projects/CR/boards/7?selectedIssue=CR-67


[CR-67]: https://curiouslearning.atlassian.net/browse/CR-67?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
